src/lib/components/railbar/index.scss

```diff
@@ -0,0 +1,35 @@
+.q-railbar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  text-align: center;
+  vertical-align: middle;
+
+  border: 0;
+  border-radius: 0;
+  margin: 0;
+  padding: 0.5rem;
+
+  position: fixed;
```

Comment:
Another minor thing: Why are there spaces? Also, the more important CSS properties should come first, so it's kinda hard to understand why `position` is below `border`.^^